bite-size-notes/week-0-b.md

```diff
@@ -0,0 +1,2 @@
+## new notes
+dslkfjsldkjfls
```

Comment:
Adding more comments here

Reply:
Incorporated this comment in the updated version.